pybossa/view/account.py

```diff
@@ -1151,6 +1151,61 @@ def add_metadata(name):
     flash("Input saved successfully", "info")
     return redirect(url_for('account.profile', name=name))
 
+
+@blueprint.route('/<user_name>/taskbrowse_bookmarks/<short_name>', methods=['GET', 'POST'])
+@login_required
+def taskbrowse_bookmarks(user_name, short_name):
+    user = user_repo.get_by_name(name=user_name)
+    if user is None:
+        return abort(404)
+    if current_user.name != user_name:
+        return abort(403)
+
+    all_bookmarks = user.info.get('taskbrowse_bookmarks', {})
+    proj_bookmarks = all_bookmarks.get(short_name, {})
+    # get all bookmarks for project
+    if request.method == 'GET':
+        return jsonify(proj_bookmarks)
+    # add a bookmark
+    if request.method == 'POST':
+        url = request.body.get('url', None)
+        name = request.body.get('name', None)
+        if name is None or len(name) > 100 or \
+            url is None or len(url) > 500:
+            abort(400)
```

Comment:
Will use get error banner with detailed error message. if not, i think it would be useful to prompt user with details about the error so that they can self troubleshoot/fix the errors. 
Also consider logging this error and any other places wherein logging would be useful to troubleshoot.

Reply:
fixed via https://github.com/bloomberg/pybossa/pull/832/commits/36a4707fe170566458061b887e5ec487b7952f31